debian/ceph-mgr.postinst

```diff
@@ -30,6 +30,8 @@ case "$1" in
 	    openssl req -new -nodes -x509 \
 		-subj "/O=IT/CN=ceph-mgr-restful" \
 		-days 3650 -keyout "$PKEY" -out "$CERT" -extensions v3_ca
+	    chown root.ceph "$CERT" "$PKEY"
```

Comment:
I thought it was supposed to be root:ceph. While '.' works as well (at least in fedora), I don't see it documented in man page.